src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -89,6 +89,7 @@
 #include "qgsjsonutils.h"
 #include "qgsjsoneditwidget.h"
 #include "qgspointcloudlayer.h"
+#include "qgscolorrampimpl.h"
```

Comment:
@nyalldawson , is this needed?

Reply:
Yes, it's used to create a colored plot in that file